Adding tags in gem

sylhare opened the pull request:
Allo,

So I have added and tested tags as a layout, it works (but you still need to add the title in the none layout for it to appear) as said in #59.
Also I have added some tags to one of the page (as an example for anyone testing the theme).
And I have updated the gemspec (removing tag, but adding search as of #60)

sylhare committed:
Updating gemspec

tags.hml will be transformed as a layout, adding search.html

sylhare committed:
Updating tags as a layout

rohanchandra commented:
Nice one @Sylhare! could you also update the docs for enabling search (in the gem version)?

sylhare committed:
Updating doc for search feature

sylhare commented:
Allo @rohanchandra, 
so I have updated/added the search documentation in the readme.
It is very simple, you only have to switch to true the `search` in the `_config.yml`. It works also as in the gem since we added the `search.html` file into our gem (which is only used to display the results).

rohanchandra commented:
Great, thanks for that 👍
